lib/ak_page_creator.rb

```diff
@@ -1,3 +1,24 @@
-class AkPageCreator
+class AkPageCreator < AkCreator
+  SUPPORTED_PAGE_TYPES = {petition: 'petition', donation: 'donation'}
 
+  def create_page(name, title, language, url, type)
+    if type == SUPPORTED_PAGE_TYPES[:donation]
```

Comment:
I'd make this into a switch or sth if we'll have more than two supported page types